### PR TITLE
165145288 Users Should Be Able To Delete Articles 

### PR DESCRIPTION
**Pull request by chikeozulumba**
#### Description
The code base was initially written in `ES6/ES7 Javascript` specs and NodeJS doesn't support it. Hence `Babel` would be used to transpile it down to `ES5 Javascript` which NodeJS running both locally and on `Heroku` - the production environment - supports.
#### Type of change
- [ ] Bug Fix (a non-breaking change which fixes an issue)
- [x] New Feature (a non-breaking change that adds functionality)
- [ ] New Chore (a non-breaking change that adds functionality)
- [ ] Breaking Change (a fix or feature which would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update
#### How Has This Been Tested?
- [x] End to End
- [x] Integration
#### Checklist:
- [x] Modify migration 
- [x] Add auth middleware in routes
- [x] Improve tests coverage
#### PT
[#165145288](https://www.pivotaltracker.com/story/show/165145288)

### CODE REVIEW

File: .sequelizerc [hunk 1]
```diff
@@ -2,7 +2,7 @@ const path = require('path');
 
 module.exports = {
   "config": path.resolve('./src/config', 'databaseConfig.js'),
-  "models-path": path.resolve('./src/db_api/models'),
+  "models-path": path.resolve('./src/models'),
   "seeders-path": path.resolve('./src/seeders'),
   "migrations-path": path.resolve('./src/migrations')
 };
```
Comment (Simplemart17) [inline .sequelizerc]: You might want to look into adding newline here.

File: src/middlewares/authenticator.js [hunk 2]
```diff
@@ -19,33 +19,35 @@ class Authenticator {
...
       [str, token] = req.get('Authorization').split(' ');
     } else {
       token = req.get('Authorization') ? req.get('Authorization') : req.headers.token;
     }
     if (!token) {
       return res.status(401)
         .json(responseFormat({
-          success: false,
-          data: 'No token supplied',
+          success: 'error',
```
Comment (Simplemart17) [inline src/middlewares/authenticator.js]: I think this is meant to be `status` and not `success`. You might want to look into it.

File: src/middlewares/authenticator.js [hunk 3]
```diff
@@ -19,33 +19,35 @@ class Authenticator {
...
+          message: 'No token supplied',
         }));
     }
     jwt.verify(token, process.env.JWTKEY, (error, decodedToken) => {
       if (error) {
         return res.status(401)
-          .json(responseFormat({ success: false, data: 'Invalid token supplied' }));
+          .json(responseFormat({ success: 'error', message: 'Invalid token supplied' }));
```
Comment (Simplemart17) [inline src/middlewares/authenticator.js]: I think this also is meant to be `status` and not `success`

File: tests/integration/user.js [hunk 4]
```diff
@@ -71,18 +71,18 @@ describe('Signup Authentication Test', () => {
...
+  //     .end((err, res) => {
+  //       expect(res).to.have.status(500);
+  //       expect(res.body)
+  //         .to.have.property('status')
+  //         .eql('error');
+  //       expect(res.body).to.have.property('message')
+  //         .eql('Something Went Wrong');
+  //       done();
+  //     });
+  // });
```
Comment (Simplemart17) [inline tests/integration/user.js]: This could be better taken off if not useful Thanks

**Review by Temmyogunbo — APPROVED** [review]

**Pull request merged**